schema/types/isolator.go

```diff
@@ -24,6 +24,7 @@ var (
 	isolatorMap map[ACIdentifier]IsolatorValueConstructor
 
 	ErrIncompatibleIsolator = errors.New("isolators set contains incompatible types")
+	ErrInvalidIsolator      = errors.New("invalid isolator")
```

Comment:
Yeah - in this case in particular I think it helps because as a consumer of the package it's unclear why I'd ever encounter this (it's unfortunate that we allow them to even get into this situation)